cmd/fleet/serve.go

```diff
@@ -475,9 +503,9 @@ the way that the Fleet server works.
 			if _, err := startIntegrationsSchedule(ctx, instanceID, ds, logger); err != nil {
 				initFatal(err, "failed to register integrations schedule")
 			}
-			if config.MDMApple.Enable {
-				startAppleMDMDEPProfileAssigner(ctx, instanceID, config.MDMApple.DEP.SyncPeriodicity, ds, depStorage, logger, config.Logging.Debug)
-			}
+			//if config.MDMApple.Enable {
+			//	startAppleMDMDEPProfileAssigner(ctx, instanceID, config.MDMApple.DEP.SyncPeriodicity, ds, depStorage, logger, config.Logging.Debug)
```

Comment:
Is the plan to re-add these before we cut the next release?

Reply:
See https://github.com/fleetdm/fleet/pull/8730#discussion_r1028063878, I'll uncomment and keep the prototype code active instead, until we're ready to fully replace it.